server/db-mongoose.js

```diff
@@ -0,0 +1,19 @@
+// FIXME: keep file only if using mongodb
+const mongoose = require("mongoose");
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.29.1/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module 'mongoose'.